packages/kujua-sms/tests/kujua-sms/utils.js

```diff
@@ -160,294 +162,6 @@ exports['getLabels'] = function(test) {
     test.done();
 };
 
-exports['getValues no clinic'] = function(test) {
```

Comment:
If you're going to remove the tests then should we remove the function too?  Do you think it could be useful to keep or is it duplicated somewhere?  https://github.com/medic/medic-webapp/blob/develop/packages/kujua-sms/kujua-sms/utils.js#L395